views/present.erb

```diff
@@ -7,6 +7,7 @@
     <link rel="icon" type="image/x-icon" href="/assets/images/favicon.ico">
     <link rel="stylesheet" href="/assets/css/madetech-frontend.css">
     <link rel="stylesheet" href="/assets/css/style.css">
+     <meta http-equiv="refresh" content="30">
```

Comment:
```suggestion
     <meta http-equiv="refresh" content="3600">
```

Reply:
resolved in https://github.com/madetech/deliveries/pull/38/commits/ec8c70626b79fc4f49eeb207969ec388afae435d